src/libs/LocalePhoneNumber.js

```diff
@@ -39,6 +39,10 @@ Onyx.connect({
  * @returns {String}
  */
 function formatPhoneNumber(number) {
+    if (!number) {
+        return '';
```

Comment:
I think this is a good approach, but I'm unsure if it's going to be okay, in all uses, for it to be an empty string.

Reply:
I originally had it to just return `number`, but my thinking was that if we're passed an incorrect falsey value (like `undefined` or `0` or something), it would be better to just return an empty string.

Reply:
Yeah totally. What I'm saying is what will the caller of the function do with `''`? Hopefully it's fine, but it's used all over the place so how do we verify that it works?

Reply:
At least it definitely won't crash if we return `''`, since the caller expects a string.